lib/install/management/appliance.go

```diff
@@ -63,6 +63,15 @@ const (
 	volumeStoresID = "VolumeStores"
 )
 
+//VCH Inventory Constants
+const (
+	alreadyExistsError            = "An object already exists on the inventory path for vch (%s) that is not an inventory folder"
+	invalidNameError              = "An invalid name was specified for the VCH: %s does not meet the naming criteria for a vch"
+	unexpectedInventoryFaultError = "unexpected fault when attempting to create the inventory folder %s please see vic-machine.log for more information"
```

Comment:
remove inventory

Reply:
This is only used when folder creation fails.